dist/js/bootstrap-datepicker.js

```diff
@@ -6,11 +6,11 @@
 
 (function(factory){
     if (typeof define === 'function' && define.amd) {
-        define(['jquery'], factory);
+      define(['jquery'], factory(jQuery || window.jQuery || require('jquery')));
     } else if (typeof exports === 'object') {
-        factory(require('jquery'));
+      factory(jQuery || window.jQuery || require('jquery'));
```

Comment:
Hi @jdnierth
This line does not throw an error in a Node/Commonjs environment?

Reply:
*facepalm* you are correct it should throw an error in Node.js environment since there is no `window` object, hence reverted this line. Thanks for the nudge!